### PR TITLE
Oy2 19468 - Updates to SCSS to match Figma Designs & HCD Feedback

### PR DESCRIPTION
**Pull request by jdinh8124**
## Purpose
We needed to override certain button designs with scss to match figma and overall Macpro color patterns 

#### Pull Request Creator Checklist

- [x] It is documented what type of request this is.
- [x] There is a detailed description for the purpose of this PR.
- [x] This PR is in line with the intention of this Project

### CODE REVIEW

File: .storybook/preview.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import "../src/assets/css/styles.css";
+import "../src/assets/theme/styles.scss"
```
Comment (jphechter) [inline .storybook/preview.js:2]: I'm not 100% on what this is importing. Does it make line 1 redundant?

Comment (jphechter) [inline .storybook/preview.js:2]: I'm not 100% on what specifically is covered by this import. Does this make line 1 redundant?

File: src/assets/theme/_uswds-theme-custom-styles.scss [hunk 2]
```diff
@@ -20,4 +20,59 @@ i.e.
...
+  &:active {
+    background-color: #ffffff;
+  }
+
+  &:disabled,
+  &:hover:disabled {
+    background-color: #5b616a;
+    color: black;
+  }
+}
```
Comment (jphechter) [inline src/assets/theme/_uswds-theme-custom-styles.scss:78]: Should this live in `/Button`?

**Review by jphechter — COMMENTED** [review]
Was literally just thinking about this. Thank you!

I'm happy with all this, but I do have a design question. Do we want to keep relevant component styles in the associated directory, or keep it at the global level in `styles.scss`?

File: .storybook/preview.js (hunk 1 above)
Comment (jdinh8124) [reply .storybook/preview.js:2]: The first import looks like it's the USWDS styles, the second one is for custom scss/overriding themes.

File: src/assets/theme/_uswds-theme-custom-styles.scss (hunk 2 above)
Comment (jdinh8124) [reply src/assets/theme/_uswds-theme-custom-styles.scss]: I have updated the logic, good call out

**Review by jphechter — APPROVED** [review]

**Comment by jdinh8124** [comment]
> Was literally just thinking about this. Thank you!
> 
> I'm happy with all this, but I do have a design question. Do we want to keep relevant component styles in the associated directory, or keep it at the global level in `styles.scss`?

I'm thinking keep relevant component styles in its respective directory and then forward it to the global level styles.scss. I think thats a decent pattern, but open for more suggestions

**Pull request merged**